test/network/scenarios/common.js

```diff
@@ -17,6 +17,9 @@
 const childProcess = require('child_process');
 const utils = require('../utils');
 
+const NODE_FINISHED_SYNC_REGEX = /Finished sync/;
```

Comment:
Why are we using `Finished sync` as a regular expression?

Reply:
I like doing `regex.test(...)`  instead of `string.indexOf(...) !== ...` it doesn't matter for me.

Reply:
I added a comment in the code.